core/shared/src/main/scala/scalaz/zio/duration/package.scala

```diff
@@ -0,0 +1,15 @@
+package scalaz.zio
+
+import java.util.concurrent.TimeUnit
+
+package object duration {
+
+  implicit final class DurationInt(private val n: Int) extends AnyVal with DurationConversions {
```

Comment:
These can become implicit `def` methods that wrap the int / long in `DurationSyntax`.